### DISCUSSION TITLE
v2/md-enhance/zh/guide/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#433] [Announcements]
# v2/md-enhance/zh/guide/

通过安装并启用此插件，你可以在 Markdown 中使用更多的语法。 请不用担心你的网站大小，如果你不启用相关功能，最终代码不会包含这些功能相关的代码。 同时 VuePress 本身新增了一些 Markdown 语法，如果你对它们不熟悉，可以 查看官方文档 (https://v2.vuepress.vuejs.org/zh/guide/markdown...

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/

**Comment by tsauliu** [comment]
test 评论

**Reply by tsauliu** [reply]
评论的评论
